src/App.vue

```diff
@@ -23,6 +23,58 @@ export default {
     title: 'Welcome',
     // all titles will be injected into this template
     titleTemplate: '%s | aec.works',
+    htmlAttrs: {
+      lang: 'en',
+      amp: false,
+    },
+    meta: [
+      { charset: 'utf-8' },
+      { name: 'viewport', content: 'width=device-width, initial-scale=1' },
```

Comment:
Do these tags clash with tags currently defined in `public/index.html`

Reply:
Ah yeah good point. I'll remove it from the vue-meta and leave the ones in the index.html